### DISCUSSION TITLE
docker部署alist后 为什么无法使用 IP:Port 的方法打开Alist网页

### DISCUSSION THREAD

**Discussion by Ctory-Nily** [AlistGo/alist#6839] [Q&A] [answered]
docker中已经部署好Alist
我可以通过127.0.0.1:5244来打开网页
但是我使用192.168.1.XX:5244的宿主机IP+port就无法打开网页
![image](https://github.com/user-attachments/assets/fc942f7c-e91d-4a92-9fce-50a73edfb13b)

**Comment by Ctory-Nily** [comment]
防火墙已经打开5244的端口 / 防火墙关闭也试过了 也不行
宿主机无法ping通容器内IP，不知道为啥
根据https://blog.csdn.net/weixin_39642622/article/details/110462545  的步骤来进行操作，也还是不能解决
![Uploading image.png…]()

**Comment by Ctory-Nily** [comment, marked as answer]
已解决 删除.wslconfig

**Reply by bruceshi-o** [reply]
你是windows还是Linux 我用的vps也是同样的问题

**Reply by Ctory-Nily** [reply]
win 我在wsl上安装的docker

**Reply by bruceshi-o** [reply]
> win 我在wsl上安装的docker

我Linux用docker安装也是这样 最后反代才可以用
